creme/creme_core/templates/creme_core/detail/property-type.html

```diff
@@ -31,36 +31,27 @@
 {% endblock %}
 
 {% block title_buttons %}
- {% if object.is_custom %}
-  {% with admin_perm=user|has_perm_to_admin:'creme_core' %}
+{% if object.is_custom %}
+    {% with admin_perm=user|has_perm_to_admin:'creme_core' %}
     {% if object.enabled %}
     <div class='bar-action'>
-      {% if admin_perm %}
+        {% if admin_perm %}
         <a href="{{object.get_edit_absolute_url}}" class="bar-button">
             {% widget_icon name='edit' size='brick-hat-bar-button' label=_('Edit') %}
         </a>
-      {% else %}
+        {% else %}
         <span class="bar-button forbidden is-disabled">
             {% widget_icon name='edit' size='brick-hat-bar-button' label=_('Edit') %}
         </span>
-      {% endif %}
+        {% endif %}
```

Comment:
Please keep my indentation ; it's not always very consistent or whatever, but these are the rules I try to follow:
 - indent the HTML tags with 4 spaces to indicate nesting level between them.
 - indent the template-tags to indicate nesting level (so I will use indentation with1/2/3 spaces).